test/regress/data-count-without-data-section.txt

```diff
@@ -0,0 +1,26 @@
+;;; TOOL: run-interp-spec
+(assert_malformed
+  (module binary
+    "\00asm" "\01\00\00\00"
+    "\01\04\01\60\00\00"       ;; Type section: 1 type
+    "\03\02\01\00"             ;; Function section: 1 function
+    "\05\03\01\00\01"          ;; Memory section: 1 memory
+    "\08\01\00"                ;; Start section: function 0
+    "\0c\01\01"                ;; Data Count section: 1 segment
+    "\0a\0e\01"                ;; Code section: 1 function
+
+    ;; function 0
+    "\0c\00"
+    "\41\00"                   ;; i32.const 0
+    "\41\00"                   ;; i32.const 0
+    "\41\00"                   ;; i32.const 0
+    "\fc\08\00\00"             ;; memory.init dataidx=0 memidx=0
+    "\0b"                      ;; end
```

Comment:
Do you even need a code section at all for this test?   In fact can't you just have a data count section and nothing else?

Reply:
probably not, no, the multiple fuzzing reports found that it could crash the interpreter if actual code was executed so that's how we minified the test case originally. however, we also upstreamed it as-is. would you like us to minify it further?

Reply:
ah, it has been minified further, okay: https://github.com/WebAssembly/spec/commit/7ff01f577b3cb0695bddd9083b7d09a289a3ea21